packages/react/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { type HTMLAttributes, forwardRef } from 'react';
+import cn from 'classnames';
+
+import { Box } from '../Box';
+
+import classes from './Card.module.css';
+
+export type CardProps = {
+  /**
+   * Shadow size of the card
+   * @default undefined
+   */
+  shadow?: 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge';
+  /**
+   * Border width of the card
+   * @default undefined
+   */
+  border?: 'default' | 'subtle' | 'strong';
+  /**
+   * Border radius of the card
+   * @default undefined
+   */
+  borderRadius?: 'small' | 'medium' | 'large';
```

Comment:
You should extend these props from the `Box` component, so we don't have duplicates.
Change `border` to `borderColor` in the props, and add `& Boxprops` to the end of this type